pages/mesosphere/dcos/services/cassandra/2.7.0-3.11.4/release-notes/index.md

```diff
@@ -61,97 +62,14 @@ render: mustache
 
 ## Improvements
 
-- Service names are now validated with a regex
+- Service names are now validated with a regex.
 
-# Version 2.4.0-3.0.16
+# Release Notes for {{ model.techShortName }} Service version 2.4.0-3.0.16
 
 ## Bug Fixes
 
 - Fix a bug where an out of date configuration ID would be selected when restarting or replacing pods. This could lead to configuration updates being reverted to the values with which the service was initially deployed. ([#2694](https://github.com/mesosphere/dcos-commons/pull/2694))
 
 ## Updates
 
-- Upgrade JRE to 1.8u192 to address CVEs
- -->
```

Comment:
I would like to keep all these release notes as comments. Since we're deleting all the previous versions' directories and information, at least we can keep the release notes handy so that it's easy to find for developers. We often get asked by support if version X supports feature Y for a lot of our packages and going to release notes is a quick way to determine that. If we remove this, we're forced to go on a scavenger hunt in Github.

Reply:
Cassie and I have been discussing this. Rather than delay the release of these documents while we decide the issue, I'm going to go with the release notes as is and ask @ck4adventure to merge them so we can get the docs out. We can always update the release notes later. I'll make a note to come back to this issue, because it will affect future docs releases as well. Thanks for bringing it to my attention; you raise a good point, but we need to formulate policy on this.